WalletWasabi.Fluent/Views/Login/PasswordFinder/PasswordFinderIntroduceView.axaml

```diff
@@ -20,42 +19,38 @@
           <Setter Property="DockPanel.Dock" Value="Top"/>
           <Setter Property="TextWrapping" Value="Wrap"/>
         </Style>
+
         <Style Selector="TextBlock.title">
+          <Setter Property="FontWeight" Value="Bold"/>
           <Setter Property="FontSize" Value="17"/>
```

Comment:
Shouldn't this use also heading font size style for consistency ?

Reply:
yes